model/storage/Sql.php

```diff
@@ -188,4 +196,20 @@ public function flush()
         }
         return true;
     }
+
+    /**
+     * @param string $field
+     * @param mixed $value
+     * @return float|int|string
+     */
+    private function castValue($field, $value)
```

Comment:
if possible please don't forget to unit test this

Reply:
Tests added, please review